python/graphstorm/model/lm_embed.py

```diff
@@ -406,6 +423,24 @@ class GSLMNodeEncoderInputLayer(GSNodeEncoderInputLayer):
         available.
     use_fp16 : bool
         Use float16 to store the BERT embeddings.
+
+    Examples:
+    ----------
+    .. code:: python
+        from graphstorm import get_feat_size
+        from graphstorm.model import GSgnnNodeModel, GSLMNodeEncoderInputLayer
+        from graphstorm.dataloading import GSgnnNodeTrainData
+        np_data = GSgnnNodeTrainData(...)
+        model = GSgnnNodeModel(...)
+        feat_size = get_feat_size(np_data.g, 'feat')
+        node_lm_configs = [{"lm_type": "bert",
+                        "model_name": "bert-base-uncased",
+                        "gradient_checkpoint": True,
+                        "node_types": ['a']}]
+        lm_train_nodes=10
+
+        encoder = GSLMNodeEncoderInputLayer(np_data.g, feat_size, 128, node_lm_configs, lm_train_nodes)
```

Comment:
The order of parameters are incorrect.
```suggestion
        encoder = GSLMNodeEncoderInputLayer(g=np_data.g, 
                                                                               node_lm_configs=node_lm_configs,
                                                                               feat_size=feat_size,
                                                                               embed_size=128,
                                                                               num_train=lm_train_nodes)
```